src/app/report/edit-display/edit-display.component.html

```diff
@@ -1,115 +1,90 @@
+        >Rerun
         </button>
-        <app-toggle-button
-          data-cy-report="toggleEdit"
-          class="mx-1"
-          [labels]="{on: 'Edit', off:'View'}"
-          [value]="editingEnabled"
-          (valueChanged)="toggleEditMode($event)"
-        ></app-toggle-button>
-
-        <span class="button-divider">&#124;</span>
-        @if (report.xml) {
-          <button
-            data-cy-report="rerun"
-            class="btn btn-info mr-1"
-            [title]=" editingEnabled ? 'Please save your changes to rerun the report' : 'Rerun this report'"
-            [disabled]="editingEnabled"
-            (click)="openDifferenceModal('saveRerun')"
-          >Rerun</button>
-        }
-        @if (editingEnabled) {
-          <button
-            data-cy-report="discard"
-            class="btn btn-info mr-1"
-            title="Discard Changes"
-            (click)="openDifferenceModal('discard')"
-          ><i class="fa fa-trash"></i></button>
-          <button
-            data-cy-report="save"
-            class="btn btn-info"
-            title="Save"
-            (click)="openDifferenceModal('save')"
-          ><i class="fa fa-save"></i></button>
-        }
-        @if (report.encoding === 'Base64') {
-          <button class="btn btn-info" title="Convert to Base64" (click)="changeEncoding($event)">Base64</button>
-        }
+      }
+      @if (editingEnabled) {
+        <button
+          data-cy-report="discard"
+          class="btn btn-info"
+          title="Discard Changes"
+          (click)="openDifferenceModal('discard')"
+        ><i class="fa fa-trash"></i></button>
+        <button
+          data-cy-report="save"
+          class="btn btn-info"
+          title="Save"
+          (click)="openDifferenceModal('save')"
+        ><i class="fa fa-save"></i></button>
+      }
+      @if (report.encoding === 'Base64') {
+        <button class="btn btn-info" title="Convert to Base64" (click)="changeEncoding($event)">Base64</button>
+      }
 
-        @if (report.stub) {
-          <div class="w-fit mx-1 my-0">
-            <select class="form-control" (change)="selectStubStrategy($event)">
-              <option [value]="-1" [selected]="report.stub === -1">Use report level stub strategy</option>
-              <option [value]="0" [selected]="report.stub === 0">Always stub this checkpoint</option>
-              <option [value]="1" [selected]="report.stub === 1">Never stub this checkpoint</option>
-            </select>
-          </div>
-        }
-      </div>
+      @if (report.stub) {
+        <div class="w-fit mx-1 my-0">
+          <select class="form-control" (change)="selectStubStrategy($event)">
+            <option [value]="-1" [selected]="report.stub === -1">Use report level stub strategy</option>
+            <option [value]="0" [selected]="report.stub === 0">Always stub this checkpoint</option>
+            <option [value]="1" [selected]="report.stub === 1">Never stub this checkpoint</option>
```

Comment:
Wouldn't this be better with a ngModel?

Reply:
Yes you're right, I didn't change it in this PR because it's already quite large, I've made an issue for this on the refactoring board.